taui/src/components/neighborhood-details.js

```diff
@@ -225,6 +217,7 @@ export default class NeighborhoodDetails extends PureComponent<Props> {
 
     const bestJourney = neighborhood.segments && neighborhood.segments.length
       ? neighborhood.segments[0] : null
+    console.log(bestJourney)
```

Comment:
Stray console log here :scroll:

Reply:
😬 Eek! Thanks. Force pushed it outta there.